### PR TITLE
settings page

### LINKED ISSUE #129: Feature: Create a settings page
### Description
Proposed design
image

### Checklist

Before submitting your pull request, ensure that you have completed the following tasks:

- [x] Gone through the [rules and guidelines](https://github.com/oxiton-foundation/click-metrics/issues/24) of the project.
- [x] When you are creating PR, your PR should title follows the specified format: [Feature/Enhancement/Bug Fix]: Brief description of the change.
- [x] Attached OSSI or GSSoC in the heading of the Issue.
- [x] I have checked this issue is not already covered in another issue.

### Expected Time to Complete
2 days if assigned today or tomorrow
else after June 25th (due to exams)

### PR DESCRIPTION
**Pull request by AkhilaSunesh**
fixes #129 
image

### CODE REVIEW

**Comment by Swarnendu0123** [comment]
Remove the `package-lock.json` from client folder and push a commit.

**Comment by rahulpoonia29** [comment]
@AkhilaSunesh Also utilise the ShadCn component for the UI instead of raw elements.

**Comment by AkhilaSunesh** [comment]
Alright thanks for the advice I will incorporate it shortly. Within today

File: client/src/index.css [hunk 1]
```diff
@@ -79,3 +79,57 @@
     font-feature-settings: "rlig" 1, "calt" 1;
   }
 }
+
+/* Custom CSS for the toggle switch */
```
Comment (Swarnendu0123) [inline client/src/index.css:83]: use tailwind CSS, in place of normal CSS

File: client/src/index.css [hunk 2]
```diff
@@ -79,3 +79,10 @@
     font-feature-settings: "rlig" 1, "calt" 1;
   }
 }
+
+<div class="relative inline-block w-60 h-34">
```
Comment (Swarnendu0123) [inline client/src/index.css:83]: Eid you added this intentionally?

**Comment by Swarnendu0123** [comment]
#182 addressing the same issue. But with a better approach. I will prefer #182 over this PR. Generally the settings page doesn't contain only the email and phone number.

**Pull request closed without merging**